paywall/next.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = withSourceMaps({
     }
 
     return {
-      '/': { page: '/' },
+      '/': { page: '/paywallHome' },
```

Comment:
I am not sure we need to prefix all pages with `paywall` since this is the paywall app, this is pretty obvious right?

Reply:
I literally did this because you asked me to ;). `'/'` refers to `src/pages/index.js` and that is why I was using it originally. We renamed the page from `index.js` and that forces updating both here and in `_server.js`.

Reply:
Sorry about the confusion here. I think what we need is `'/': { page: '/home' },`!